app/processing/route-debt-to-request-editor.js

```diff
@@ -0,0 +1,21 @@
+const db = require('../data')
+const { getHoldCategoryId } = require('../holds')
+const sendDebtMessage = require('../messaging/send-debt-message')
+const holdAndReschedule = require('../reschedule')
+const util = require('util')
+
+const routeToRequestEditor = async (paymentRequest) => {
```

Comment:
name should be `routeDebtToRequestEditor`

comment left around removing defunct `route-to-request-editor` file so be careful with cleaning up function names in `index.js` and tests